nibabel/tests/test_nifti1.py

```diff
@@ -1242,3 +1243,16 @@ def test_rt_bias(self):
                 # Hokey use of max_miss as a std estimate
                 bias_thresh = np.max([max_miss / np.sqrt(count), eps])
                 assert_true(np.abs(bias) < bias_thresh)
+
+
+@runif_extra_has('slow')
+def test_large_nifti1():
+    image_shape = (91, 109, 91, 1200)
+    img = Nifti1Image(np.zeros(image_shape, dtype=np.float32),
+                      affine=np.eye(4))
+    with InTemporaryDirectory():
+        img.to_filename('test.nii.gz')
+        del img
+        data = load('test.nii.gz').get_data()
+    assert_array_equal(np.asarray(image_shape), data.shape)
```

Comment:
Shouldn't these both be tuples? If one doesn't need `np.asarray`, I imagine the other doesn't, either. At which point, you could probably just do `assert_equal`.